src/odemis/model/_core.py

```diff
@@ -31,7 +31,9 @@
 import threading
 from future.moves.urllib.parse import quote
 from odemis.util import inspect_getmembers
-import resource
+
+if os.name != 'nt':
+    import resource
```

Comment:
Let's move these lines together with the (only) function that uses it. Then you can put a if block around the entire import + function, and if it's windows, just define the function as doing nothing.